distro-plugins/slackware.sh

```diff
@@ -0,0 +1,61 @@
+##
+## Plug-in for installing Slackware Arm
+##
+
+DISTRO_NAME="Slackware"
+
+# You can override a CPU architecture to let distribution
+# be executed by QEMU (user-mode).
+#
+# You can specify the following values here:
+#
+#  * aarch64: AArch64 (ARM64, 64bit ARM)
+#  * armv7l:  ARM (32bit)
+#  * i686:    x86 (32bit)
+#  * x86_64:  x86 (64bit)
+#
+# Default value is set by proot-distro script and is equal
+# to the CPU architecture of your device (uname -m).
+#DISTRO_ARCH=$(uname -m)
+
+# Returns download URL and SHA-256 of file in this format:
+# SHA-256|FILE-NAME
+get_download_url() {
+        local rootfs
+        local sha256
+        
+        case "$DISTRO_ARCH" in
+                aarch64)
+                        rootfs="ftp://ftp.arm.slackware.com/slackwarearm/slackwarearm-devtools/minirootfs/roots/slackarm-14.2-miniroot_01Jul16.tar.xz"
+                        sha256="ce9533c6920f621e23f4e379a2f7c92568807187ac88e93cbd53f9ecee2d7899"
+                        ;;
+                armv7l|armv8l)
+                        rootfs="ftp://ftp.arm.slackware.com/slackwarearm/slackwarearm-devtools/minirootfs/roots/slackarm-14.2-miniroot_01Jul16.tar.xz"
+                        sha256="ce9533c6920f621e23f4e379a2f7c92568807187ac88e93cbd53f9ecee2d7899"
```

Comment:
You can't use same rootfs for multiple architectures. It doesn't matter whether you can run 32bit software under 64bit CPU. You need to find or build rootfs for each architecture.

Also date 01jul16 seems too awkward, we are in 2021 now.